generators/app/templates/_dockerTaskGeneric.ps1

```diff
@@ -0,0 +1,103 @@
+<#
+.SYNOPSIS
+Builds and runs a Docker image.
+.PARAMETER Compose
+Runs docker-compose.
+.PARAMETER Build
+Builds a Docker image.
+.PARAMETER Clean
+Removes the image <%= imageName %> and kills all containers based on that image.
+.PARAMETER Environment
+The enviorment to build for (Debug or Release), defaults to Debug
+.EXAMPLE
+C:\PS> .\dockerTask.ps1 -Build
+Build a Docker image named <%= imageName %>
+#>
```

Comment:
This comment sets up integration with PowerShell's Get-Help feature. Attached is the basic output
[help.txt](https://github.com/Microsoft/generator-docker/files/54331/help.txt)